utils/tree/Makefile

```diff
@@ -8,12 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tree
-PKG_RELEASE:=1
 PKG_VERSION:=2.0.2
-PKG_SOURCE_URL:=http://mama.indstate.edu/users/ice/tree/src
-PKG_HASH:=7d693a1d88d3c4e70a73e03b8dbbdc12c2945d482647494f2f5bd83a479eeeaf
+PKG_RELEASE:=2
+
+PKG_SOURCE:=unix-$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://gitlab.com/OldManProgrammer/unix-$(PKG_NAME)/-/archive/$(PKG_VERSION)
+PKG_HASH:=fda48f7443c6883c599cd9e87e93fd843c1cd162e68794d23e4dc6d10efc6250
+PKG_BUILD_DIR:=$(BUILD_DIR)/unix-$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
I am thinking if you don't want to download this from [GitHub](https://github.com/Old-Man-Programmer/tree), which is a backup (as said in project's homepage: http://mama.indstate.edu/users/ice/tree/) and the repository (with the tarball) is named as tree.

Reply:
It's actually gitlab.  I am not sure how else to setup the download.  I modeled this after [apparmor](https://github.com/openwrt/packages/blob/master/utils/apparmor/Makefile).  Please suggest the edit to achieve what you're thinking.

Reply:
I think you did not understand me. From the project homepage, it is clear that he is moving code to GitLab, which he will be using as a primary repository, but there is also a backup on GitHub, and with using the GitHub, you can avoid doing this craziness. :-)

Reply:
Ah!  I understand now.  Done.

Reply:
See the comment below from @neheb regarding the filename.  I switched back to this method.

Reply:
@BKPepe - Finally got the syntax right, see latest commit :+1:

Reply:
Yeah, it seems alright. :)